src/shared/unit-file.c

```diff
@@ -199,7 +199,7 @@ static bool lookup_paths_mtime_exclude(const LookupPaths *lp, const char *path)
                streq_ptr(path, lp->runtime_control);
 }
 
-static bool lookup_paths_mtime_good(const LookupPaths *lp, usec_t mtime) {
+bool lookup_paths_mtime_good(const LookupPaths *lp, const usec_t mtime) {
```

Comment:
hmm, the `const` is not really something that is useful here... we generally don't use it on non-pointer arguments, since it doesn't actually help anyone, this argument is copied anyway on the stack and whether the function internally changes its own copy on the stack or not should be an implementation detail of the function. Please remove the "const" again

Reply:
Ok, there was a compiler warning somewhere - I'll cast instead